src/ucp/core/ucp_listener.c

```diff
@@ -246,6 +247,36 @@ ucp_listen(ucp_listener_h listener, const ucp_listener_params_t *params)
     return status;
 }
 
+static void ucp_listener_vfs_show_ip(void *obj, ucs_string_buffer_t *strb)
+{
+    ucp_listener_h listener   = obj;
+    struct sockaddr *sockaddr = (struct sockaddr*)&listener->sockaddr;
+    char ip_str[UCS_SOCKADDR_STRING_LEN];
+
+    if (inet_ntop(sockaddr->sa_family, ucs_sockaddr_get_inet_addr(sockaddr),
```

Comment:
1. please compare with `NULL` explicitly.
2. maybe, if `inet_ntop` fails, printing the errno makes sense?